modules/installation-osp-provider-network-preparation.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// TODO: Add mods to all appropriate installation assemblies after first pass is approved.
+
+[id="installation-osp-provider-network-preparation_{context}"]
+= {rh-openstack} provider network requirements for cluster installation
+
+Before you install a {product-title} cluster, your {rh-openstack-first} deployment and provider network must meet a number of conditions:
+
+* The link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/sec-networking-concepts#install-networking[{rh-openstack} networking service (Neutron) is enabled] and accessible through the {rh-openstack} networking API.
```

Comment:
I guess the same question extends to each of the external links to RHOSP docs on the Portal, throughout.

Reply:
Addressed (hopefully) in 6d8c0f8381a479ae136618c130f7bfbace23e9a5. TY!